lib/processAgent.js

```diff
@@ -123,7 +128,9 @@ const addMetrics = function (agent, pidToCheck) {
       tags: {
         token: infraToken,
         'process.type': 'child',
-        'process.name': 'node'
+        'process.name': 'node',
+        pid: pidToCheck,
+        ppid: masterProcess.ppid
       },
       fields: {
         count: Math.max(processes.length - 1, 0) || 0
```

Comment:
This whole section needs to be moved inside the `childProcesses.forEach` loop. Also, `processCountMetric` on line 112 needs to have the `pid` and `ppid` from the `masterProcess` object.

Reply:
The keys need to be `process.pid` and `process.ppid`.
So:

```js
'process.pid': masterProcess.pid,
'process.ppid': masterProcess.ppid
```

in both `processChildCountMetric` and `processCountMetric`.